README.md

```diff
@@ -23,6 +23,11 @@ NB: The lambda function created assumes that the Amazon EKS cluster's Kubernetes
 enabled, if your endpoint only has private access enabled then you must modify the `template.yml` file to ensure the 
 lambda function is running in the correct VPC and subnet.
 
+This lambda can also be used against a non-EKS kubernetes cluster, reading a kubeconfig file from an S3 bucket
```

Comment:
Let's add a note here to stress that access to the bucket should be as restricted as possible, this is a very import piece of config that we don't want falling into the wrong hands.